bbs/newuser.cpp

```diff
@@ -170,13 +171,26 @@ void input_name() {
   } while (!ok && !a()->sess().hangup());
 }
 
+// Function to check if both first and last names are entered
+bool containsFirstAndLastName(const std::string& name) {
+    std::istringstream stream(name);
```

Comment:
This could be simplified as:

change name to not be a reference
wwiv::strings::StringTrim(&name)
return wwiv::strings::contains(name, ' ');